mmcv/transforms/processing.py

```diff
@@ -400,3 +406,688 @@ def __repr__(self):
         repr_str += f'pad_val={self.pad_val}), '
         repr_str += f'padding_mode={self.padding_mode})'
         return repr_str
+
+
+@TRANSFORMS.register_module()
+class CenterCrop(BaseTransform):
+    """Center crop the image and segmentation mask. If the crop area exceeds
+    the original image and ``pad_mode`` is not None, then pad the original
+    image first.
+
+    Required Keys:
+
+    - img
```

Comment:
Can this module also support boxes and keypoints?

Reply:
It's not mentioned in design document but sounds reasonable. I advise to support it in a new PR.

Reply:
Sure, we can do that.